zebra-consensus/src/transaction.rs

```diff
@@ -831,6 +850,36 @@ where
                 );
             }
 
+            // # Consensus
+            //
+            // > The Spend transfers and Action transfers of a transaction MUST be
+            // > consistent with its vbalanceSapling value as specified in § 4.13
+            // > ‘Balance and Binding Signature (Sapling)’ on p. 49.
```

Comment:
Wouldn't it be better to leave out the page number here? The section is already on p. 50 now.
```suggestion
            // > ‘Balance and Binding Signature (Sapling)’.
```